src/libs/actions/Report.js

```diff
@@ -1038,23 +1038,48 @@ function deleteReportComment(reportID, reportAction) {
  * @param {Number} reportID
  */
 function openReport(reportID) {
-    const sequenceNumber = getMaxSequenceNumber(reportID);
-    API.write('OpenReport',
+    let onyxData;
+    let commonOptimisticData = {
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
+        key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+        value: {
+            lastVisitedTimestamp: Date.now(),
+            unreadActionCount: 0,
+        }
+    };
+    // only show loading state if the report has not been open yet
```

Comment:
This seems like something the view should handle. We can still set the loading state. The view will decide to use it or not based on whether it has initial report actions to display.